src/commands/lambda/flare.ts

```diff
@@ -99,6 +103,17 @@ export class LambdaFlareCommand extends Command {
       }
     }
 
+    // Validate start/end flags if both are specified
+    let startMillis
+    let endMillis
```

Comment:
```suggestion
    let startMillis, endMillis
```

Reply:
ESLint doesn't let us do that in this repo